packages/server/lib/modes/interactive.ts

```diff
@@ -170,6 +173,36 @@ export = {
       makeGraphQLServer(),
     ])
 
+    const clearCtxAndQuit = async () => {
+      try {
+        await clearCtx()
```

Comment:
There is a comment saying we should not call this in runtime: https://github.com/cypress-io/cypress/pull/22606/files#diff-1827debd8088884005aba0e14a94e1e6815ceb1ad567db970f9f327d9feb7c9dR27

If we are doing this, I think we need to change the comment?

Also, maybe a comment here explaining why we clearCtx before closing? Could link to the relevant GH issue